src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/CondaBomTool.groovy

```diff
@@ -56,7 +56,7 @@ class CondaBomTool extends BomTool {
     public boolean isBomToolApplicable() {
         def containsFiles = detectFileManager.containsAllFiles(sourcePath, 'environment.yml')
         if (containsFiles) {
-            condaExecutablePath = executableManager.getPathOfExecutable(ExecutableType.CONDA, detectConfiguration.getCondaPath())
+            condaExecutablePath = getExecutablePath(ExecutableType.CONDA, true, detectConfiguration.getCondaPath())
```

Comment:
Is this supposed to be `findExecutablePath(....)` not `getExecutablePath(...)` ?